folly/tracing/AsyncStack.cpp

```diff
@@ -19,6 +19,7 @@
 #include <atomic>
 #include <cassert>
 #include <mutex>
+#include <exception>
```

Comment:
Nit: this should probably be alphabetized after "cassert" and before "mutex"

Reply:
This will be handled automatically internally - we have clang-format with include sorting enabled there, though I'm not sure if we have the configs in the folly repo itself.